packages/client/src/open-feature.ts

```diff
@@ -17,6 +17,11 @@ const GLOBAL_OPENFEATURE_API_KEY = Symbol.for('@openfeature/web-sdk/api');
 type OpenFeatureGlobal = {
   [GLOBAL_OPENFEATURE_API_KEY]?: OpenFeatureAPI;
 };
+type NameProviderTuple = {
+  namespace?: string;
```

Comment:
The namespace PR hasn't been approved yet.

Reply:
I think it will be and I just thought it would be an easy thing to miss and a rename, and not a problem to use it early here, but I can change it to `name`.